subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/VfsRetentionHelper.java

```diff
@@ -28,11 +29,11 @@ public static void waitForChangesToBePickedUp() throws InterruptedException {
     }
 
     public static String getEnableVfsRetentionArgument() {
-        return systemProperty(VFS_RETENTION_ENABLED_PROPERTY, true);
+        return "--" + StartParameterBuildOptions.WatchFileSystemOption.LONG_OPTION;
     }
```

Comment:
Would be nice to add a method similar to `systemProperty()`, like:

```java
return booleanBuildOption(StartParameterBuildOptions.WatchFileSystemOption, true)
```

Reply:
Added.